CONTRIBUTING.md

```diff
@@ -271,7 +271,7 @@ Here is a template of what your commit message should look like:
 <BLANK LINE>
 <footer>
 ```
-
+[Back to TOC](#table-of-contents)
```

Comment:
style: note that 2 line breaks are enforced in this file before each header. There are other changes in this patch that do not respect this rule.